capsules/src/mcp230xx.rs

```diff
@@ -528,9 +524,13 @@ impl hil::i2c::I2CClient for MCP230xx<'a> {
     }
 }
 
-impl hil::gpio::Client for MCP230xx<'a> {
-    fn fired(&self, bank_number: usize) {
+impl gpio::ClientWithValue for MCP230xx<'a> {
+    fn fired(&self, value: u32) {
+        if value != 0 && value != 1 {
```

Comment:
could be simplified to `if value < 2 { break; }`

Reply:
Better inequality but I prefer return because it's more explicit that it means "stop now."